2023/das-wg-charter.html

```diff
@@ -180,15 +180,14 @@ <h2 id="scope">
           </li>
           <li>preventing the screen or system from turning off
           </li>
-          <li>monitoring the presence of nearby objects without physical
-          contact
+          <li>accessing nearby devices without physical contact
           </li>
-          <li>monitoring the ambient light level of the device’s environment
+          <li>responding to ambient light levels of the device’s environment
           </li>
-          <li>monitoring changes in motion and orientation of the hosting
+          <li>reacting to changes in motion or orientation of the hosting
           device
           </li>
-          <li>obtaining and monitoring the location of the hosting device
+          <li>providing location information, and not necessarily location information of the hosting device
```

Comment:
This section seems significantly better to me. The phrasing here could be slightly different if you want, "providing location information, either of the hosting device or of a location of interest"

Reply:
I would want to hear from chairs on this point, but Anssi marked with thumb-up, so I count that provided alternative is better.